docs/quickstart_simple.md

```diff
@@ -44,9 +44,9 @@ Open the package.json file in a text editor and update the dependencies to inclu
 
 ```JSON
   "dependencies": {
-    "@oceanprotocol/contracts": "^0.5.6",
-    "@oceanprotocol/lib": "^0.6.5",
-    "web3": "^1.3.0"
+    "@oceanprotocol/contracts": "^0.5.6", // TODO: replace version
```

Comment:
this is so wrong, forcing us to constantly update version number in here. The standard to tell people to install things in js ecosystem is NOT to tell people to edit the `package.json` but using the `npm install` command.

In this case the instructions should be:

```
npm install @oceanprotocol/lib web3
```

And that's it.